packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -33,28 +39,65 @@ class RootNodeSchema extends builder.object("root-item", {
 	child: builder.optional(ChildNodeSchema),
 }) {}
 
-function createLocalSharedTree(id: string): TreeView<typeof RootNodeSchema> {
-	const sharedTree = SharedTree.create(
+const initialState = {
+	child: {
+		propertyOne: 128,
+		propertyTwo: {
+			itemOne: "",
+		},
+		propertyThree: new Map([["numberOne", 1]]),
+	},
+};
```

Comment:
TO get errors here instead of where this is used:
```suggestion
} satisfies NodeBuilderData<RootNodeSchema>;
```

Reply:
Updated with suggestion